crates/chia-sdk-test/src/simulator.rs

```diff
@@ -286,3 +298,54 @@ impl Simulator {
         self.height += 1;
     }
 }
+
+// currently in mempool_manager.py
```

Comment:
Need to figure out a better way to do this probably

Reply:
uhh feel free to change